README.md

```diff
@@ -376,6 +376,18 @@ jobs:
           run-shadow-scans: false                              # Run SonarQube scans on all 3 platforms (next, sqc-eu, sqc-us)
 ```
 
+### Caching Configuration
+
+By default, Poetry caches `.cache/pypoetry` directory. You can customize this behavior:
```

Comment:
```suggestion
By default, Poetry caches `<poetry-cache-dir>` directory. You can disable the caching with:
```

Or just remove the "Caching Configuration" part, as there's nothing very useful here, and it's missing poetry-cache-dir.